test/FactorialServiceSpec.js

```diff
@@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
     
     it('should null if an invalid param was given', function() {
         expect(FactorialService.calculate(-1)).toBe(null);
-        expect(FactorialService.calculate("1")).toBe(null);
```

Comment:
Yeah, but this caused a problem with number float. How is done parse to int, if to write 4.5, the code sends only 4, generating result 24... which theoretically should not occur. Anyway. Thanks for the answers.